broker/src/main/java/org/apache/rocketmq/broker/transaction/queue/TransactionalMessageBridge.java

```diff
@@ -346,4 +346,16 @@ public MessageExt lookMessageByOffset(final long commitLogOffset) {
     public BrokerController getBrokerController() {
         return brokerController;
     }
+
+    public boolean escapeMessage(MessageExtBrokerInner messageInner) {
+        PutMessageResult putMessageResult = this.brokerController.getEscapeBridge().putMessage(messageInner);
+        if (putMessageResult != null
```

Comment:
how about use `org.apache.rocketmq.store.PutMessageResult#isOk` to verify whether success,
It's a little bit cleaner and it's consistent with everything else

Reply:
Good idea~ Changed as you advised.